G/GCCBootstrap/build_tarballs.jl

```diff
@@ -0,0 +1,107 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GCCBootstrap"
+version = v"9.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    # crosstool-ng will provide the build script
+    ArchiveSource("http://crosstool-ng.org/download/crosstool-ng/crosstool-ng-1.25.0_rc1.tar.bz2",
+                  "8a839df71bea7b2c411447b41b917f2142482df171733ddb92d615cec2f0f43a"),
+
+    # We provide some configs for crostool-ng
+    DirectorySource("./bundled"),
+
+    # crosstool-ng can download the files, but we'd rather download them ourselves
+    FileSource("http://mirrors.kernel.org/gnu/gcc/gcc-9.4.0/gcc-9.4.0.tar.xz",
```

Comment:
Why these are FileSources instead of ArchiveSources? crosstools-ng expects the unpacked archives?

Reply:
Yes, precisely.  crosstool-ng wants to unpack and patch the sources itself.